Sources/SourceKit/clangd/ClangLanguageServer.swift

```diff
@@ -18,7 +18,7 @@ import LanguageServerProtocolJSONRPC
 import Foundation
 
 /// A thin wrapper over a connection to a clangd server providing build setting handling.
-final class ClangLanguageServerShim: LanguageServer {
+final class ClangLanguageServerShim: LanguageServer, ToolchainLanguageServer {
```

Comment:
What are we still using from `LanguageServer` here?  Does it make sense to inline it?  If we're still using the `MessageHandler` implementation I think it will just forward everything to the `SourceKitServer` anyway, right? In which case we can just set that as the message handler, maybe?

Reply:
Yep it was just for the `MessageHandler` implementation, but that sounds right - I'll try that

Reply:
Seems to work, swapped over